enterprise_access/apps/content_assignments/migrations/0023_historicallearnercontentassignment_is_assigned_course_run_and_more.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.15 on 2024-08-26 14:06
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('content_assignments', '0022_allocated_at_not_null'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='historicallearnercontentassignment',
+            name='is_assigned_course_run',
+            field=models.BooleanField(default=False, help_text='Whether the content_key corresponds to a course run. If True, the content_key should be a course run key.'),
+        ),
+        migrations.AddField(
+            model_name='historicallearnercontentassignment',
+            name='parent_content_key',
+            field=models.CharField(blank=True, db_index=True, help_text='The globally unique content identifier of the parent content to assign to the learner.  Joinable with ContentMetadata.content_key in enterprise-catalog.', max_length=255, null=True),
+        ),
+        migrations.AddField(
+            model_name='learnercontentassignment',
+            name='is_assigned_course_run',
+            field=models.BooleanField(default=False, help_text='Whether the content_key corresponds to a course run. If True, the content_key should be a course run key.'),
+        ),
+        migrations.AddField(
+            model_name='learnercontentassignment',
+            name='parent_content_key',
+            field=models.CharField(blank=True, db_index=True, help_text='The globally unique content identifier of the parent content to assign to the learner.  Joinable with ContentMetadata.content_key in enterprise-catalog.', max_length=255, null=True),
+        ),
+        migrations.AlterField(
+            model_name='historicallearnercontentassignment',
+            name='allocated_at',
+            field=models.DateTimeField(blank=True, default=django.utils.timezone.now, help_text='The last time the assignment was allocated. Cannot be null.'),
+        ),
```

Comment:
[inform] This migration is unrelated to changes I made in this PR; seems like it's a migration that was missing (i.e., auto-generated during `makemigrations`).